assisted-inject-sample-android/src/main/java/com/example/Greeter.java

```diff
@@ -8,8 +8,8 @@
   private final String exclamation;
 
   @AssistedInject
-  Greeter(@Assisted String greeting, @Exclamation String exclamation) {
-    this.greeting = greeting;
+  Greeter(@Assisted String greetg, @Exclamation String exclamation) {
```

Comment:
is this rename intentional? :thinking:

Reply:
Nope, must have slipped in from testing.